src/lib.rs

```diff
@@ -8,11 +9,11 @@ mod scope_manager;
 mod utils;
 mod vm_core;
 
+use cairo_runner::PyCairoRunner;
 use pyo3::prelude::*;
-use vm_core::PyVM;
 
 #[pymodule]
 fn cairo_rs_py(_py: Python, m: &PyModule) -> PyResult<()> {
-    m.add_class::<PyVM>()?;
```

Comment:
If we're no longer exposing `PyVM` then we can probably just use `VirtualMachine` internally. This will make the code more efficient as it reduces interaction with the Python runtime.

Reply:
We still need PyVM to act as a PyClass so that we can use it to create PyMemory, PyMemorySegmentManager among other pyclasses which use a VM reference internally